.github/workflows/data_update.yml

```diff
@@ -15,39 +15,58 @@ jobs:
     - name: Checkout Main repo
       uses: actions/checkout@v2
       with:
-        key: ${{ secrets.data_daily_update }}
```

Comment:
I think this might have been an accident — we’ve been getting warnings about it. For example: https://github.com/sfbrigade/stop-covid19-sfbayarea/actions/runs/116699791

Reply:
Ahh, it should be "ssh-key". I will fix it. Thanks for catching.

Reply:
Oh, there is already ssh-key. Just removing the key line should be okay.